OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -803,7 +800,7 @@ void SaveHotkey()
 			Game.Settings.Save();
 
 			foreach (var hd in modData.Hotkeys.Definitions)
```

Comment:
This needs to be done inside `Set` for the first hokey. The motivation for all of my requests is that the `SettingsLogic` should not have *anything* to do with updating the duplicate flags!

Reply:
Understood. I forgot about this one. Will fix it 👍